File: Task-1/Replace_MinMax.py

size = int(input("Number of elements: "))       
numbers = input("Enter numbers seperated by spaces: " ).split()
numbers = [int(i) for i in numbers[:size]]
min_index = 0
max_index = 0
for i in range(1,len(numbers)):
    if numbers[i] < numbers[min_index]:
         min_index = i
    if numbers[i]> numbers[max_index]:
         max_index = i
numbers[min_index] , numbers[max_index] = [numbers[max_index],numbers[min_index]]
print(numbers)

File: Task-1/Replace_MinMax.cpp
#include <iostream>
using namespace std;
int main()
{
    int size;
    cin >> size;
    int ar[size];
    for(int i=0;i<size;i++)
    {
        cin >> ar[i];
    }
    int min_index = 0;
    int max_index = 0;
    int max = ar[min_index];
    int min = ar[max_index];
    for(int i=0;i<size;i++)
    {
        if(ar[i]<ar[min_index])
        {
            min_index = i;
            min = ar[i];
        }
        if (ar[i]>ar[max_index])
        {
            max_index = i;
            max = ar[i];
        }
        
    }
    ar[min_index] = max;
    ar[max_index] = min;
    for(int i=0;i<size;i++)
    {
        cout<<ar[i];
        cout<<" ";
    } 
    return 0;
}